packages/lesswrong/components/form-components/CoauthorsListEditor.tsx

```diff
@@ -10,7 +10,7 @@ const coauthorsListEditorStyles = (theme: ThemeType): JssStyles => ({
   root: {
     display: 'flex',
     marginLeft: 8,
-    marginTop: -4,
+    marginTop: -12,
```

Comment:
This was the only form component that had `hideHeader` set to true, and there was some special styling based on this in FormGroupLayout to reduce the padding in this case. I've moved that to this file (with unfortunate negative margin) so avoid having to handle `hideHeader` in the layout component